frontend/projects/upgrade/src/app/core/feature-flags/feature-flags.service.ts

```diff
@@ -17,6 +22,18 @@ export class FeatureFlagsService {
   isLoadingFeatureFlags$ = this.store$.pipe(select(selectIsLoadingFeatureFlags));
   allFeatureFlags$ = this.store$.pipe(select(selectAllFeatureFlagsSortedByDate));
   isAllFlagsFetched$ = this.store$.pipe(select(selectIsAllFlagsFetched));
+  selectSearchString$ = this.store$.pipe(select(selectSearchString));
+  selectSearchKey$ = this.store$.pipe(select(selectSearchKey));
+  selectFeatureFlagSortKey$ = this.store$.pipe(select(selectSortKey));
+  selectFeatureFlagSortAs$ = this.store$.pipe(select(selectSortAs));
+
+  selectSearchFeatureFlagParams(): Observable<Record<string, unknown>> {
```

Comment:
we should have a SearchFeatureFlagParams type instead of any or `Record<string, unknown>`.